nautobot/extras/models/tags.py

```diff
@@ -47,6 +71,19 @@ def to_csv(self):
         return (self.name, self.slug, self.color, self.description)
 
 
+class TaggableManagerField(TaggableManager):
+    """
+    Helper class for overriding TaggableManager formfield method
+    """
+
+    def formfield(self, form_class=DynamicModelMultipleChoiceField, **kwargs):
```

Comment:
I'm actually not sure DynamicModelMultipleChoiceField is the best UI experience here, as it becomes very unwieldy when selecting more than three or four distinct values - we've had similar complaints about the use of StaticSelect2Multiple (the widget for DynamicModelMultipleChoiceField) in the reskinned Admin UI.

Reply:
What alternative are you suggesting I use?

Reply:
Possibly just a standard Django `ModelMultipleChoiceField` with its default `SelectMultiple` widget? I'm not entirely sure, this may need some exploration.